source/extensions/filters/network/redis_proxy/conn_pool_impl.h

```diff
@@ -102,14 +103,18 @@ class InstanceImpl : public Instance {
     Envoy::Common::CallbackHandle* host_set_member_update_cb_handle_{};
     std::unordered_map<std::string, Upstream::HostConstSharedPtr> host_address_map_;
     std::string auth_password_;
+    std::list<Upstream::HostSharedPtr> created_hosts_;
```

Comment:
These are temporary hosts created during redirection, right? Can you name this something more descriptive?

Reply:
I'll rename it created_via_redirect_hosts_